cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/IndexManager.java

```diff
@@ -255,7 +255,7 @@ public String ensureIndexed(List<Object> fieldNames, String indexName) {
      *  @param indexType The type of index (json or text currently supported)
      *  @return name of created index
      */
-    public String ensureIndexed(List<Object> fieldNames, String indexName, String indexType) {
```

Comment:
I had planned to deprecate the old String versions of the methods and leave them for compatibility. Any thoughts on that?